examples/evm-to-evm-fungible-transfer/src/transfer.ts

```diff
@@ -16,7 +16,7 @@ if (!privateKey) {
 }
 
 const SEPOLIA_CHAIN_ID = 11155111;
-const AMOY_CHAIN_ID = 84532;
+const BASE_SEPOLIA_CHAIN_ID = 84532;
```

Comment:
did we deprecate amoy?

Reply:
no, AMOY is not correct name used here, this chain id was of base sepolia